interface/forms/procedure_order/common.php

```diff
@@ -1188,6 +1315,78 @@ class="col-form-label col-md-2"><?php echo xlt('History Order'); ?></label>
+                                                   value=''>
+                                        <?php endif; ?>
+                                        <div class='input-group-prepend'>
+                                            <button type="button" class='btn btn-secondary btn-search' title='<?php echo xla('Click to use procedure code from code popup'); ?>'>
+                                            </button>
+                                            <input type='hidden' name='form_procedure_type[<?php echo $i; ?>]' value='' />
+                                            <input type='text' name='form_proc_type_desc[<?php echo $i; ?>]'
+                                                   value=''
+                                                   title='<?php echo xla('Click to select the desired procedure'); ?>'
+                                                   placeholder='<?php echo xla('Click to select the desired procedure'); ?>'
+                                                   class='form-control c-hand sel-proc-type' readonly />
+                                            <!-- the configuration type id -->
+                                            <input type='hidden' name='form_proc_type[<?php echo $i; ?>]' value='-1' />
+                                        </div>
+                                    </td>
+                                    <td class='diagnosis-div input-group'>
+                                        <div class='input-group-prepend'>
+                                            <span class='btn btn-secondary input-group-text'>
+                                                <i class='fa fa-search fa-lg search-current-diagnoses' title='<?php echo xla('Click to search past and current diagnoses history'); ?>'></i>
+                                            </span>
+                                        </div>
+                                        <input class='form-control c-hand add-diagnosis-sel-related' type='text'
+                                               name='form_proc_type_diag[<?php echo $i; ?>]'
+                                               value=''
+                                               title='<?php echo xla('Click to add diagnosis for this test'); ?>'
+                                               readonly />
+                                    </td>
+                                    <td>
+                                        <!-- MSIE innerHTML property for a TABLE element is read-only, so using a DIV here. -->
+                                        <div class="table-responsive qoe-table-sel-procedure" id='qoetable[<?php echo attr($i); ?>]'>
+                                            <?php
+                                            $qoe_init_javascript = '';
+                                            echo generate_qoe_html($ptid, $formid, null, $i);
+                                            if ($qoe_init_javascript) {
```

Comment:
this seems odd, since `$qoe_init_javascript` is set to nothing 2 lines above and then never set to anything after that

Reply:
it gets set in the question html if memory serves.

Reply:
just realized this is from the old code in this script, so no need to address this

Reply:
@sjpadgett , correct on the generate_qoe_html since $qoe_init_javascript is used as a global